### DISCUSSION TITLE
Website UI

### DISCUSSION THREAD

**Discussion by NitinSingh0** [rushiii3/KETS#26] [General]
Why is the navigation bar removed on desktop interfaces and replaced with a hamburger menu for navigation?

**Comment by NitinSingh0** [comment]
Where should the language translation button be optimally placed on the website?

**Reply by Aaditya18SP** [reply]
### we can try something like this for language translation

1. ![image](https://github.com/rushiii3/KETS/assets/107549181/a92ec93a-2f10-4f98-af5c-1bc9099466fe)

2. ![image](https://github.com/rushiii3/KETS/assets/107549181/cdf31955-cd44-4b03-90e4-4dca5a5c7088)
3. https://simplelocalize.io/blog/posts/ui-design-language-selector-examples/

**Comment by Aaditya18SP** [comment]
> Why is the navigation bar removed on desktop interfaces and replaced with a hamburger menu for navigation?

To keep the home page UI uncluttered as possible.

**Reply by NitinSingh0** [reply]
While the hamburger menu can reduce clutter, quick access, according to me, is crucial, especially on a college website. Retaining the navigation bar ensures immediate access to essential links like admissions and academics without extra clicks, enhancing user experience.

**Reply by Aaditya18SP** [reply]
I understand your point. But a hamburger menu allows for more flexibility. We can add more tabs easily and we it will still remain uncluttered. 

***Example***
![image](https://github.com/rushiii3/KETS/assets/107549181/601e5216-eec6-4924-9ae1-76feba3f9d34)

**Reply by NitinSingh0** [reply]
To address this, we can effectively utilize the "More" section in our navigation bar to seamlessly incorporate any new tabs. Given that we likely won't need to add many new tabs, the "More" section will easily accommodate one or two additional tabs without any issue.
